Paradise threatened (#1)
LazerLit/endless-sky: data/human/Republic Paradise Threatened.TXT

# Paradise Threatened

mission "RN Analyze the Nukes"
	Name "Analyze the Nukes"
	description "Go to <destination> to drop the sensor readings off at Lovelace Labs to be analyzed."
	deadline
	source "Geminus"
	destination "Ada"
	on offer
		conversation
			`Amelia is engrossed in some kind of spreadsheet when you barge into her office and drop a datachip of scans from your fight with the Wolf Pack onto her desk. Her brows knit at this breach of protocol but she switches her display to show the data and you watch closely as her eyes widen and the colour drains from her face.`

> 			`	“Is this… this can’t be right!" she says, flipping back and forth through the images. “Nukes? I... I almost believed them!"`

Reviewer: ```suggestion
			`	"Is this... this can’t be right?!" she says, flipping back and forth through the images. “Nukes? I... I almost believed them!"`
```
I don't know enough about Commander Melgrew, but I feel like levelheadedness, even for something like nukes, is probably necessary for someone in the Navy.

Author: I think what I was going for here was to really drive home to the player just how big a deal this is. You’re right that Shah is generally unflappable but the use of nukes is so far outside of her expectations that it shakes her.

Of course it may be that this isn’t necessary because it will be the second time that nukes are used, after the Gemini’s (and the other planet I can’t remember right now) bombings.

Reviewer: Hahahaha, it just shows how much the bombings really don't have an impact, which is something I'm trying to help solve. The other planet is Martini.

			`	You remain quiet and let her process what she is seeing. It takes a while and so you make yourself comfortable on a recliner in the corner, just answering the occasional question when she wants a bit more context. Eventually she leans back on her chair shaking her head. You notice that she seems to have aged in the short time you have been here.`

> 			`	“I need a little time to decide what to do. Please stay nearby, I'm going to need your help.“`

Reviewer: ```suggestion
			`	"I need a little time to decide what to do. Please stay nearby, I'm going to need your help."`

```

> 			`	You nod and leave her alone in the office staring at the ceiling and go for a walk around the campus. The grounds of the headquarters buildings have been beautifully landscaped and it isn't difficult to find a nice green, quiet space to think.`

Reviewer: It would be better if the player was given an option to go to the gardens, or walk through the campus. I'm a sucker for funding out what 31st century architecture looks like, so describe that as well.

> 			`	However you had felt about the Free Worlds before, it is clear now that they are worse than terrorists. Their protestations of innocence in the bombings of Geminus and Martini are obviously nothing more than a cynical attempt at political obfuscation. You try to distract yourself with cargo manifests while you wait, but the events of the past few days keep exploding back into your mind. Finally you get a prompt to return to the office where Commander Shah is waiting for you.`

Reviewer: Don't tell the player what they are feeling. I'd say this is the golden rule of writing in Endless Sky. 

Now I don't know if it's possible (because I really don't know what is going on right now), but you can better convey this if you have two characters with those viewpoints, and the player has to make a choice on who they agree with more. Not only is this avoiding that "telling what the player feels" problem, you can play off those characters and bring to light the different perspectives and personalities of each side. An example of this in the Free Worlds story is Alondo and the whole Council against Tomek.

Because this seems to be one of the branches where you've picked a person to go with and the other has been sidelined, it would be harder for us to do this. The Free Worlds story has the advantage of the Council, but I don't know about Paradise Threatened.

Author: Yeah, this is a good idea. I could introduce her superior, Colonel Kent, and have them discuss it.

> 			`	“We can’t make any mistakes here," she says, “this is too important. This has to be passed up to the top, but we need to have the answers to every question that might be asked ready. I’ll talk to Anton but I already know what he’ll say, we need to get the data independently analysed and verified. We can’t risk broadcasting this, the data has to be physically taken to Ada, to Lovelace Labs. The fewer people who are in the loop the better.`

Reviewer: ```suggestion
			`	"We can’t make any mistakes here," she says, "this is too important. It has to be passed up to the top, but we need to have the answers to every question that might be asked ready. I’ll talk to Anton but I already know what he’ll say, we need to get the data independently analysed and verified. We can’t risk broadcasting this, the data has to be physically taken to Lovelace Labs; the fewer people who are in the loop, the better.`
```
This sounds very "and then, and then, this is, this is." It's too monotonous, even though I have the feeling she is talking very fast?

			`	"<first>, you of all people know what is riding on this. Can you deliver the data, and make sure that it gets to the right hands at Lovelace Labs? I can’t send anyone from the Navy without drawing attention."`

> 			`	You nod, understanding the importance of this task.`

Reviewer: Probably better to have an choice here agreeing or asking a question.

Author: Agreed.

	on complete
		dialog `As soon as you contact control you are told to land on a reserved pad some way from the main space port. When you land an aide is awaiting you and reverently loads the logs into a reinforced data case. She then tells you that they will contact you when the analysis is done, it will probably take a few days.`
	on complete
		event "Analysis Complete"

event "Analysis Complete" 5
		

mission "Return to Ada"
	minor
	to offer
		has "event: Analysis Complete"
	on offer
		dialog
			`A message pings on your console. "The analysis is complete, Captain <last>. Please return to Ada as soon as possible."`
		log "The analysis of the nukes used in Denebola is complete. Time to find out where they came from."
	destination "Ada"

mission "Investigate the Source A"
	autosave
	landing
	name "Investigate the Source"
	description "Track down the source of the plutonium used for the Free Worlds nukes."
	deadline
	passengers 1
	priority
	source "Ada"
	infiltrating
	destination "Lodestone"
	to offer
		has "Return to Ada: done"

> 	blocked `A message comes through from Commander Shah, "Captain <last>, thank you for sticking with us. I have just received word that you will have a passenger for the next part of the mission so please make sure you have room for them."`

Reviewer: ```suggestion
	blocked `A message comes through from Commander Shah. "Captain <last>, thank you for sticking with us. I have just received word that you will have a passenger for the next part of the mission, so please make sure you have room for them."`

```

	on offer
		conversation

> 			`As soon as you land a transport pulls up outside of the hatch of your ship and a small contingent of technicians start unloading equipment, supervised by a man in a generic-looking business suit. The man notices you watching and approaches, pulling out a badge identifying him as an agent of Republic Intelligence.`

Reviewer: ```suggestion
			`As soon as you land a transport pulls up outside of the hatch of your ship, and a small contingent of technicians start unloading equipment, supervised by a man in a generic-looking business suit. The man notices you watching and approaches, pulling out a badge identifying him as an agent of Republic Intelligence.`
```

			choice
				`	"Republic Intelligence? Isn't that an oxymoron?"`
				`	"What's this all about?"`
					goto friendly

> 				`	Wait for him to introduce himself.`

Reviewer: ```suggestion
				`	(Wait for him to introduce himself.)`
```

					goto friendly

> 			`	The agent stares blankly at you and as he does so you start to notice just how much muscle is bulking up his jacket. He doesn't blink and eventually you have to look away.`

Reviewer: ```suggestion
			`	The agent stares blankly at you, and as he does so, you start to notice just how much muscle is bulking up his jacket. He doesn't blink, and eventually you have to look away.`
```

Author: Commas! Commas are my nemesis! I usually put too many in my writing and it breaks up the flow too much, so I tend to comb back through what I've written and remove most of them. Now you're putting them back in! Aaarrgghhh!
Anyway, I'm going to leave most of the commas out for a little while until I can read over it all again when it's not fresh in my brain.

			goto friendly
				label friendly
			`	"Captain <last>?" the man asks, studying your face intently.`
			`	"I am Agent Norris, I have been assigned to this case by Parliament. As you can imagine they are very interested in getting results as quickly as possible."`
			choice

> 				`	Nod.`

Reviewer: ```suggestion
				`	(Nod.)`

```

				`	"As am I."`
				`	"I work faster alone."`
			`	He turns away and waves the technicians past you onto the ship.`

> 			`	"They will sweep the vessel for bugs," he says, "it won't take long. As soon as they are done we can be on our way. I will brief you as we go."`

Reviewer: ```suggestion
			`	"They will sweep the vessel for bugs," he says. "It won't take long; as soon as they are done we can be on our way, and I will brief you as we go."`

```

		log `Agent Norris of Republic Intelligence will tag along as we search for the source of the Wolf Pack nukes.`

> 	on enter "Aldebaran"
> 		conversation
> 			`As soon as you break atmosphere Agent Norris is at your shoulder.`
> 			`	"The analysis seems to confirm a Syndicate origin," he says grimly. "There are only a few worlds with plutonium mines where they would have had a chance of avoiding Republic inspectors. Our mission is to get soil samples form each of them."`

Reviewer: This shows up if you have entered the system, not landed on the planet. So you're not breaking atmosphere.
```suggestion
	on enter "Aldebaran"
		conversation
			`As soon as you enter <system>, Agent Norris is at your shoulder.`
			`	"The analysis seems to confirm a Syndicate origin," he says grimly. "There are only a few worlds with plutonium mines where they would have had a chance of avoiding Republic inspectors. Our mission is to get soil samples form each of them."`
```

Reviewer: Actually, I can't say this is right. Don't worry about it for now.

			choice
				`	"What if the Syndicate don't want to let us?"`
				`	"Sounds easier said than done."`
			`	"We can't afford any delay here, we need to shut this down before they can build up an arsenal. We will be landing directly at each of the mines and moving on as soon as we have the sample. We will not be asking permission."
			choice
				`	"Sounds good to me."`
					goto end

> 				`	"I don't want to take on the entire syndicate fleet! In my experience ignoring traffic control makes planetary militias trigger-happy."`

Reviewer: ```suggestion
				`	"In my experience ignoring traffic control makes planetary militias trigger-happy."`
```
I think this is far too long.

			`	"That's why I'm here," Agent Norris says. "We will be carrying out a warrant. The only way they could legally stop us would be to file an injunction with the Supreme Court, and we're not going to give them time to do that."
			label end
			`	He points to a system on your nav screen.`
			`	"Our first stop is <destination>, Captain. Let's get moving."`
	on complete
		conversation

> 			`Your descent to the surface of <destination> is violently bumpy, the turbulence threatening to send the <ship> hurtling out of control. As soon as it becomes clear that you are not heading towards the small landing site, your com system lights up with warnings.`

Reviewer: ```suggestion
			`Your descent to the surface of <destination> is violently bumpy, the turbulence threatening to send the <ship> hurtling out of control. As soon as it becomes clear that you are not heading towards the small landing site, your comm system lights up with warnings.`
```
'Comm' for 'communications' reads better than 'com'.

			`	"Uh, <ship>, please be advised that you are in restricted airspace. Divert to the heading indicated immediately."`
			`	Agent Norris steps calmly forward and presses the button to respond.

> 			`	"<destination> control," he says, his voice smooth as silk, completely unlike his usual gravelly rumble. "This is <ship>, currently acting on parliamentary orders. I am transmitting a warrant which allows us to act as we see fit. I understand that you will want to check it with your superiors, so be my guest."`

Reviewer: ```suggestion
			`	"<destination> planetary spaceport control," he says in a voice smooth as silk, completely unlike his usual gravelly rumble. "This is <ship>, currently acting on Parliamentary orders. I am transmitting a warrant which allows us to act as we see fit. I understand that you will want to check it with your superiors, so be my guest."`
```

			`	"That's it, Captain," he says as soon as he has taken off the breath mask. "Break atmosphere and we can move on to the next location."`

mission "Investigate the Source B"
	autosave
	landing
	name "Investigate the Source"
	description "Track down the source of the plutonium used for the Free Worlds nukes."
	deadline
	passengers 1
	priority
	source "Lodestone"
	destination "Placer"
	to offer
		has "Investigate the Source A: done"
	on enter
		conversation
			`It is a relief to be back in space where you are significantly less likely to be hit by a lava bomb. Agent Norris calls up the star chart and selects the next destination.`
			`	"<destination> next, Captain," he says. "We should expect a little more pushback this time as the warrant won't be a complete surprise. Don't worry though, they can't get an injunction this quickly."`
			`	"So how quickly can they get one?" you ask. "And how many more systems are we looking at?"`
			`	He looks at you for several long moments, his face expressionless.
			`	"I won't lie to you, it will be tight."
			choice
				`	"Fair enough, this was never going to be easy."`

> 					end

Reviewer: For this conversation to end, it needs to go to a label with nothing in it.

```suggestion
					goto end
```

				`	"This is sounding worse by the moment."`
				`	"And if they get an injunction? What, we just stop?"`
			`	For the first time since you met him Agent Norris actually smiles, a grim expression which makes you think that he really hopes the worst will happen.`
			`	"Captain <last>," he says, leaning casually against a console. "We will complete this mission. The whole mission, not just the easy parts. The Syndicate are not above the law."`

> 			`	He nods once and retires to his bunk leaving you to plot the course to <system>, his words still echoing in your head. You think you've seen a fair bit of the galaxy, much more than most of humanity, and in your experience in Syndicate space the Syndicate makes the laws. This will certainly be interesting.`

Reviewer: ```suggestion
			`	He nods once and retires to his bunk leaving you to plot the course to <system>, his words still echoing in your head. You think you've seen a fair bit of the galaxy, much more than most of humanity, and in your experience in Syndicate space the Syndicate makes the laws. This will certainly be interesting.`
			label end
```

	on complete
		conversation

> 			`You start your descent, aiming for a small but well-organized settlement in the southern hemisphere. It takes a little longer than usual for traffic control to contact you, but when they finally do you realize that is probably because they were getting somebody senior on the line.`

Reviewer: ```suggestion
			`You start your descent, aiming for a small but well-organized settlement in the southern hemisphere. It takes a little longer than usual for spaceport control to contact you, but when they finally do you realize that it's probably because they were getting somebody senior on the line.`
```

	npc
		personality disables plunders harvests staying
		system "Algol"
		fleet "Large Core Pirates"
	to offer
		has "Investigate the Source B: done"
	on offer
		conversation
			`Agent Norris collects his sample and you make a short hop to the spaceport to refuel. The ground crew does seem a little more disorganized than usual but Agent Norris goes out to bully them into getting their work done as quickly as possible while you take the opportunity to do some quick maintenance. Soon enough <ship> is fully fueled and you seal the hatch for takeoff.
			`	"Where to this time?" you ask.`
			`	"<destination>," Norris says simply. He turns to go back to his bunk but just as he does so the same voice as before comes over the channel to <source> control.
			`	"<ship>, please advised, we have received warnings of increased pirate activity in the region, recommend delaying departure until it has been dealt with."`
			`	Norris simply gestures for you to take this.
			choice
				`	"A bluff?"`
				`	"What do you think?"`
				`	"They'll try anything, won't they?"`
			`	He shrugs. "You know the stakes as well as I do, Captain <last>. I see this as a good sign, they're desparate."`

> 			`	You clear your throat and then key the comm. "Thank you <source> control. We will be careful."`

Reviewer: ```suggestion
			`	You clear your throat and then key the comm. "Thank you, <source> control. We will be careful."`
```

	on enter "Al Dhanab"
		dialog
			`It looks like the warning from <source> control about pirates was real. As you jump emerge from hyperspace you are confronted by a small group who are already plundering.`
	on enter "Algol"
		conversation
			`You emerge in <system> to find a large fleet of pirates waiting. Agent Norris swears, looking flustered for the first time since you met him.`
			`	"This can't be coincidence!" he growls. "A pirate fleet this large in a small system like <system>? It makes no sense. Somebody has paid them off!"`
			choice
				`	"Don't worry, that's why they have me ferrying you around."`

> 				`	"They might be strong but <ship> is faster!"`

Reviewer: ```suggestion
				`	"They might be strong, but the <ship> is faster!"`
```

> 				`	"As long as we can make it into the atmosphere I can evade them."`

Reviewer: ```suggestion
				`	"As long as we can make it into the atmosphere, I can evade them."`
```
Haven't you already taken off?

Author: This one is (supposed to be) upon entering the destination system, they encounter a blockade but need to make it to the planet.

			`	Agent Norris looks at you and nods, and this might just be your imagination but you think you see a hint of respect in his eyes.`
	on complete
		conversation

> 			`You make it to the surface of <destination> and land at the test site. While Agent Norris is outside taking the sample you get a message through from <destination> control once again offering you free fueling and the run of the spaceport amenities. How nice of them.`

Reviewer: ```suggestion
			`You make it to the surface of <destination> and land at the test site. While Agent Norris is outside taking the sample, you get a message from <destination> control once again offering you free fueling and the run of the spaceport amenities. How nice of them.`
```
I've got to mention that free refueling is weird since every captain seems to be able to refuell for free, and they probably wouldn't advertise this. Unless you really want them to look desperate.

		system "Ruchbah"
		fleet "Syndicate Extremists"
	npc
		personality heroic
		government "Syndicate (Extremist)"
		system "Bellatrix"
		fleet "Large Syndicate"
	to offer
		has "Investigate the Source C: done"
	on offer
		conversation
			`You hop over to the spaceport and as you do so you can see from your scope that there are no pirates left in the system. As before the ground crew seem unusually incompetent, and as before Agent Norris goes outside to deal with it. You take the time to perform some routine maintenance, and then you notice a ruby red laser light shining on one of the consoles. You freeze. Are they goind to try to assassinate you? You're not aware of any handheld munition which could penetrate the armor of your ship but suddenly you wonder what the labs on Hephaestus might have quietly come up with.`
			`	Before you can respond the light starts moving, faster and faster until it is spelling out a message.`
			`	"1,000,000 credits to seal your hatch and power down your sensors"`
			choice

> 				`	Do as they wish. A million credits is a million credits.`
> 				`	Darken the viewports and ignore the message.`
> 					end

Reviewer: ```suggestion
				`	(Do as they wish. A million credits is a million credits.)`
				`	(Darken the viewports and ignore the message.)`
					goto end

```

> 				`	You press the button to seal the outer hatch and then, after a moments hesitation, power the sensors down completely. As you sit in your chair you wonder what is going on outside but the chime of your comm which tells you that one million credits has been added to your account dampens down your curiosity significantly.`

Reviewer: ```suggestion
				`	You press the button to seal the outer hatch and then, after a moment's hesitation, power the sensors down completely. As you sit in your chair you wonder what is going on outside, but the chime of your comm which tells you that one million credits has been added to your account dampens down your curiosity significantly.`
```

> 				`	After what seems like an age but was apparently only five minutes, a green laser spot appears on your console and just as quickly disappears again. You take that as an 'all clear' and look out your viewport. The ground crew are still working but there is no sign of Agent Norris. You open the hatch and take a look around but the agent is nowhere to be seen.`

Reviewer: ```suggestion
				`	After what seems like an age but was apparently only five minutes, a green laser spot appears on your console and just as quickly disappears again. You take that as an 'all clear' and look out your viewport. The ground crew are still working, but there is no sign of Agent Norris. Just to be sure, you open the hatch and take a look around, but the agent is nowhere to be seen.`
```
Those last two lines are repetitve.

				`	Without knowing what other systems you are supposed to be investigating you can't continue the mission. You tell yourself that the truth about the Wolf Pack nukes has to come out eventually, and in the mean time at least you have a million more reasons to sleep well.`
					payment 1000000
					log "Accepted a bribe to look the other way instead of tracing the source of the Wolf Pack nukes. The money was just too good to turn down."
					fail
	on enter
		conversation
			`Agent Norris instructs you to plot a course for <destination> and seems completely unaware of the odd message that you received.
			choice

> 				`	Tell him what happened.`
> 				`	No need to worry him.`

Reviewer: ```suggestion
				`	(Tell him what happened.)`
				`	(There's no need to worry him.)`
```

					goto onward
				`	You decide that he should know and explain that somebody used a laser to send you a message asking you to seal the hatch and deactivate the sensors. Agent Norris nods slowly.
				`	"What did they offer you?" he asks as though he is aksing what you had for your lunch.
				`	"A million credits," you say.`
				`	He nods again looking mildly impressed. "And you turned it down? Captain <last>, I didn't know you cared."`
				`	"Wait, was that a joke?" you ask, but Norris' face is still stony.`
				`	"Thank you, Captain," he says. "If you had accepted I have no doubt I would be dead by now. That or under interrogation. We must proceed more carefully from now on. If we can avoid refueling on any Syndicate worlds we should do so."`
				label onward
					`	You program the nav computer and prepare to jump, but before you can do so Agent Norris receives a message and raises a hand as though asking for you to wait. He reads the message several times before looking back up at you.`
					`	"The injunction has been approved," he says. "The Syndicate won't be so friendly from here on out."`

> 					`	You laugh. If that was the Syndicate being 'friendly' you wouldn't want to see them hostile, but it looks like you're going to.`

Reviewer: I wouldn't laugh in this situation, the Syndicate is directly willing to kill an agent of Republic Intelligence, which seems more serious than just some old merchant getting plundered by a pirate. 

You shouldn't tell the player what they are feeling.

> 					`	"<destination> is the last planet on our list," Norris tells you quietly. "We just need to get to the test site and then get the samples back to Ada."`

Reviewer: ```suggestion
					`	"<planet> is the last planet on our list," Norris tells you quietly. "We just need to get to the test site and then get the samples back to Ada."`

```

					choice
						`	"Piece of cake."`
						`	"No Syndicate ship built with parts from the lowest bidder is going to stop the <ship>."`
						`	"I guess it's too late to back out now."`
	on enter "Bellatrix"
		conversation
			`As soon as you emerge into the system you are hailed by a large Syndicate fleet.`
			`	"<ship>, we have reason to believe that you are carrying illegal cargo. Heave to and prepare to be boarded."`
			choice
			`	"Not today!"`

> 			`	"That's a bogus charge and you know it! We are conducting an investigation and if you try to obstruct it you will be the one breaking the law!"`
> 			`	Say nothing, let your flying do the talking.`
> 			`	Comply with their instructions.`

Reviewer: ```suggestion
			`	"That's a bogus charge and you know it!"
			`	"We are conducting an investigation, and if you try to obstruct it you will be the one breaking the law!"`
			`	(Say nothing, let your flying do the talking.)`
			`	(Comply with their instructions.)`
```

				goto surrender
			label surrender
				`	Agent Norris sees what you are doing and comes storming over.`
				`	"We must complete this mission!" he hisses.
				choice
					`	"There's too many of them!"`

> 						goto actually surrender

Reviewer: ```suggestion
						goto "actually surrender"
```

					`	"Hold on, I have an idea!"`

> 						# I want to be able to lure the fleet closer with this option so that it's easier to make a run to the planet, although I'm not sure if that's possible.

Reviewer: It's not possible.

> 						label actually surrender
> 							`	Agent Norris shakes his head in disgust and turns away from you... and then bursts into motion and before you can react he is on top of you and with one jerk of his arms you lie on the deck of your ship with your neck broken.`

Reviewer: ```suggestion
						label "actually surrender"
							`	Agent Norris shakes his head in disgust and turns away from you... and then before you can react he is on top of you, and with one jerk of his arms you lie on the deck of your ship with your neck broken.`
```

							die

> 			`	You continue on a course for <destination> hoping that the Syndicate admiral is bluffing. The seconds tick by and just when you think they are going to back down a voice comes over the comm.`
> 			`	"If that's the way you want it..."`
> 			`	Immediately your sensors indicate that the Syndicate fleet has started targetting you. You are going to have to do this the hard way.`
> 				end

Reviewer: ```suggestion
			`	You continue on a course for <destination>, hoping that the Syndicated Security officer is bluffing. The seconds tick by, and just when you think they are going to back down, a voice comes over the comm.`
			`	"If that's the way you want it..."`
			`	Immediately your sensors indicate that the Syndicate fleet has started targetting you. You are going to have to do this the hard way.`
				label end
```

	on complete
		conversation

> 			`You make it to the surface and as you are coming in to land you can see that a small unit of soldiers are setting up some defences. You screech over them and open up with your weapons, sending them sprawling in all directions. Agent Norris retrieves a rifle from his bunk, and when he sees you looking at it he shrugs.`
> 			`	"My technicians put it in when they were sweeping the ship."`

Reviewer: ```suggestion
			`You make it to the surface, and as you are coming in to land you can see that a small unit of Syndicated Security soldiers are setting up some defences. You screech over them and open up with your weapons, sending them sprawling in all directions.`
			`	Agent Norris retrieves a rifle from his bunk, and when he sees you looking at it he shrugs. "My technicians put it in when they were sweeping the ship."`
```

> 			`	As he surges through the hatch moving faster than any human has a right to you can't help but wonder what else those technicians were doing. After you drop Norris on Ada you decide that you'll take some time to sweep <ship> yourself.`
> 			`	There is the sound of gunfire from outside and smoke surrounds the ship. Your sensors show four human heat signatures running and bent low but you can't tell which one is Norris. They all seem to be converging on the clear area in front of the mine but it looks like none of them can see any of the others. You watch as one spins from cover, sweeping his weapon from side to side. Just then one of the others jerks, is lifted from the ground and then falls down dead! What happened?`

Reviewer: ```suggestion
			`	As he surges through the hatch, moving faster than any human has a right to, you can't help but wonder what else those technicians were doing. After you drop Norris on Ada, you decide that you'll take some time to sweep <ship> yourself.`
			`	There is the sound of gunfire from outside, and smoke surrounds the ship. Your sensors show four human heat signatures running and bent low but you can't tell which one is Norris. They all seem to be converging on the clear area in front of the mine, but it looks like none of them can see any of the others. You watch as one spins from cover, sweeping his weapon from side to side. Just then, one of the others jerks, lifts from the ground and then falls down dead, all in a flash.`
```

> 			`	A second person enters the clearing and then exchanges hand signals with the first. They both start moving in paralel, creeping slowly and each covering their own side. You look for the fourth person and find that he is now lying dead too. Before you can figure out what is going on the two remaining figures turn and start firing behind them, but what their aiming at is a mystery as nothing shows on your scope. First one and then the other expend their ammo and pause... and then beams of fire erupt from in front of them, taking both down before they have a chance to react.`
> 			`	When the smoke eventually clears you see Norris calmly toting the sample back towards the ship.`
> 			`	"Back to Ada, Captain," he says when he is back on board. You are not about to argue with him.`

Reviewer: ```suggestion
			`	A second person enters the clearing and then exchanges hand signals with the first. They both start moving in parallel, creeping slowly and each covering their own side. You look for the fourth person and find that he is now lying dead too. Before you can figure out what is going on, the two remaining figures turn and start firing behind them, but what their aiming at is a mystery as nothing shows on your scope. First one and then the other expend their ammo and pause... and then beams of fire erupt from in front of them, taking both down before they have a chance to react.`
			`	When the smoke eventually clears you see Norris calmly toting the sample back towards the ship.`
			`	"Back to Ada, Captain," he says when he is back on board. You are not about to argue with him.`
```
